kubernetes/zenko/templates/cosmos-operator/crd.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.cosmos.enabled }}
```

Comment:
Should we really make cosmos optional? I think it should just be like backbeat... (deployed even if unused)

Reply:
Mostly for the reason that it creates privileged accounts. If someone didn't want a service running that has/creates this kind of access, it could be turned off